.github/workflows/update-iso-image-versions.yml

```diff
@@ -0,0 +1,71 @@
+name: "update-iso-image-versions"
+on:
+  workflow_dispatch:
+env:
+  GOPROXY: https://proxy.golang.org
+  GO_VERSION: '1.21.3'
+permissions:
+  contents: read
+jobs:
+  update-all:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+      - uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe
+        with:
+          go-version: ${{env.GO_VERSION}}
+          cache-dependency-path: ./go.sum
+      - name: Bump versions
+        id: bumpVersions
+        run: |
+          make update-buildkit-version
+          make update-cni-plugins-version
+          make update-containerd-version
+          make update-cri-o-version
```

Comment:
maybe we should keep cri-o out of it ...

Reply:
Removed cri-o